binfmt/binfmt_unloadmodule.c

```diff
@@ -161,6 +161,32 @@ int unload_module(FAR struct binary_s *binp)
           file_munmap(binp->mapped, binp->mapsize);
         }
 
+#ifdef CONFIG_ARCH_USE_SEPARATED_SECTION
+  for (i = 0; binp->sectalloc[i] != NULL && i < binp->nsect; i++)
```

Comment:
add one level indent

Reply:
Is it better to keep the indentation consistent with the above?

Reply:
like line 144